lib/airbrake/railtie.rb

```diff
@@ -12,13 +12,16 @@ class Railtie < ::Rails::Railtie
 
     initializer "airbrake.middleware" do |app|
 
-      middleware = if defined?(ActionDispatch::DebugExceptions)
-        # Rails >= 3.2.0
-        "ActionDispatch::DebugExceptions"
-      else
-        # Rails < 3.2.0
-        "ActionDispatch::ShowExceptions"
-      end
+      middleware =
+        if !::Rails.env.production? && defined?(::BetterErrors::Middleware)
```

Comment:
Is there any good reason why we can't remove the `if !::Rails.env.production?` test and only check `if defined?(::BetterErrors::Middleware)`?